Library/Homebrew/extend/os/mac/hardware/cpu.rb

```diff
@@ -63,6 +63,14 @@ def universal_archs
         [arch_64_bit, arch_32_bit].extend ArchitectureListExtension
       end
 
+      # True when running under an Intel-based shell via Rosetta on an
+      # Apple Silicon Mac. This can be detected via seeing if there's a
+      # conflict between what `uname` report and the underlying `sysctl` flags,
+      # since the `sysctl` flags don't change behaviour under Rosetta.
+      def running_under_rosetta?
```

Comment:
```suggestion
      def in_rosetta?
```

Or similar? Not huge on the current name but don't have strong objections.

Reply:
`running_under_rosetta?` seemed a bit long to me too, but I didn't have a better suggestion. I like `in_rosetta?`.